caffe2/proto/caffe2.proto

```diff
@@ -98,10 +98,12 @@ message Argument {
   optional string name = 1;
   optional float f = 2;
   optional int64 i = 3;
-  optional bytes s = 4;
-  repeated float floats = 5;
-  repeated int64 ints = 6;
-  repeated bytes strings = 7;
+  optional string s = 4;
+  optional bytes b = 5;
+  repeated float floats = 6;
+  repeated int64 ints = 7;
+  repeated string strings = 8;
+  repeated bytes bytes = 9;
```

Comment:
Can you elaborate on this? The bytes field is now called `s`, which in my opinion should be used for string. I can move `optional string s = 4;` to the end to preserve order, but I would really prefer to rename `bytes s` to `bytes b`.